packages/components/src/core/styles/common/defaultTheme.ts

```diff
@@ -557,4 +633,5 @@ export interface Borders {
   warning: Border;
 }
 
-export const defaultTheme = createThemeAdaptor(defaultThemeOptions);
+export const defaultTheme = (theme: "light" | "dark") =>
```

Comment:
Actually, we should preserve the `defaultTheme` and use `light` mode so that the API doesn't cause breaking changes to consumers of the library. We can export a `theme` variable that is the function to determine which one to choose.

Reply:
Done!